third-party/projects/alloy-ui/src/aui-autosize/js/aui-autosize-iframe.js

```diff
@@ -309,6 +309,7 @@ var AutosizeIframe = A.Component.create({
                 instance._iframeHeight = value;
 
                 instance.node.setStyle('height', value);
+                instance.node.scrollIntoView(true);
```

Comment:
This change affects other places where the aui-autosize-iframe is being used, being the embedded layout one of them. Are we 100% sure we want to scrollintoview for all of them? Would not be this better handled via opt-in configuration? @markocikos your thoughts are welcome

Reply:
hello @dsanz @markocikos , if i understood you correctly, the term "handled via opt-in" is that we should create a new attribute and let outside control the behavior of the iframe. is that correct?
i put a draft code in my repo for this matter, please help to correct me
https://github.com/chileces/liferay-frontend-projects/commit/193f70bd10547b070d34b33d79790e0e990b523e

Reply:
I like the idea of adding this to API, as a new attribute. Looking at https://github.com/chileces/liferay-frontend-projects/commit/193f70bd10547b070d34b33d79790e0e990b523e, I'd make two changes:
- `true` probably makes sense as the default, I would expect that to be the most common desired behavior. 
- In addition to boolean, I would allow object as an argument, as it is allowed in [the native scrollIntoView](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView).

Reply:
i pushed the changes according to your comments, please check if it is ok.

Reply:
@chileces yes, that's the idea. This way, DXP client code manages the flag instead of us producing a behavioral change across all usages of the component (note that our customers may be using this in their third party developments too).